From 3062db0b48d7f2167f60712c302a6a0144f82f70 Mon Sep 17 00:00:00 2001
From: Yonit Halperin <yhalperi@redhat.com>
Date: Sun, 1 Apr 2012 10:00:29 +0300
Subject: [PATCH 2/3] controller: add COLOR_DEPTH and DISABLE_EFFECTS messages

rhbz #787447

Signed-off-by: Yonit Halperin <yhalperi@redhat.com>
---
 spice/controller_prot.h |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/spice/controller_prot.h b/spice/controller_prot.h
index a60b122..f7b1f26 100644
--- a/spice/controller_prot.h
+++ b/spice/controller_prot.h
@@ -75,6 +75,9 @@ enum {
 
     CONTROLLER_ENABLE_SMARTCARD,
 
+    CONTROLLER_COLOR_DEPTH,
+    CONTROLLER_DISABLE_EFFECTS,
+
     //spice client -> extrenal app
     CONTROLLER_MENU_ITEM_CLICK = 1001,
 };
-- 
1.7.9.3

