From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Frediano Ziglio <fziglio@redhat.com>
Date: Tue, 14 Mar 2017 11:55:28 +0000
Subject: [spice-protocol] Add error reports from server to guest

Acked-by: Christophe de Dinechin <cdupontd@redhat.com>
---
 spice/stream-device.h | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/spice/stream-device.h b/spice/stream-device.h
index 0185fbd..67d70e1 100644
--- a/spice/stream-device.h
+++ b/spice/stream-device.h
@@ -83,6 +83,8 @@ typedef enum StreamMsgType {
     STREAM_TYPE_DATA,
     /* server ask to start a new stream */
     STREAM_TYPE_START_STOP,
+    /* server notify errors to guest */
+    STREAM_TYPE_NOTIFY_ERROR,
 } StreamMsgType;
 
 /* Generic extension capabilities.
@@ -144,4 +146,22 @@ typedef struct StreamMsgStartStop {
     uint8_t codecs[0];
 } StreamMsgStartStop;
 
+/* Tell guest about invalid protocol.
+ * This message is sent by the host to the guest.
+ * The server will stop processing data from the guest.
+ *
+ * States allowed: any
+ */
+typedef struct StreamMsgNotifyError {
+    /* numeric error code.
+     * Currently not defined, set to 0.
+     */
+    uint32_t error_code;
+    /* String message, UTF-8 encoded.
+     * This field terminate with the message.
+     * Not necessary NUL-terminated.
+     */
+    uint8_t msg[0];
+} StreamMsgNotifyError;
+
 #endif /* SPICE_STREAM_DEVICE_H_ */
