From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Frediano Ziglio <fziglio@redhat.com>
Date: Thu, 17 Aug 2017 10:26:29 +0100
Subject: [spice-protocol] Add support for mouse movement

Signed-off-by: Frediano Ziglio <fziglio@redhat.com>
Acked-by: Christophe de Dinechin <cdupontd@redhat.com>
---
 spice/stream-device.h | 11 +++++++++++
 1 file changed, 11 insertions(+)

diff --git a/spice/stream-device.h b/spice/stream-device.h
index d8612a4..bf919ed 100644
--- a/spice/stream-device.h
+++ b/spice/stream-device.h
@@ -87,6 +87,8 @@ typedef enum StreamMsgType {
     STREAM_TYPE_NOTIFY_ERROR,
     /* guest cursor */
     STREAM_TYPE_CURSOR_SET,
+    /* guest cursor position */
+    STREAM_TYPE_CURSOR_MOVE,
 } StreamMsgType;
 
 /* Generic extension capabilities.
@@ -196,5 +198,14 @@ typedef struct StreamMsgCursorSet {
     uint8_t data[0];
 } StreamMsgCursorSet;
 
+/* Guest cursor position
+ * This message is sent by the guest to the host.
+ *
+ * States allowed: Streaming
+ */
+typedef struct StreamMsgCursorMove {
+    int32_t x;
+    int32_t y;
+} StreamMsgCursorMove;
 
 #endif /* SPICE_STREAM_DEVICE_H_ */
